### DISCUSSION TITLE
Issue with the preview of html content

### DISCUSSION THREAD

**Discussion by aniket-tarento** [Sunbird-Ed/Community#247] [General Discussion]
I'm trying to upload html zip, the course is getting uploaded but the preview of the content is not fitting properly in the html player. It is coming with the scroll bars and zoomed content. Is there is any limitation on the dimensions of the html content that we need to follow to get proper preview of the content? attaching the screenshot of the content player.
![html-player-1](https://user-images.githubusercontent.com/98937737/201847143-d580c707-1051-4f81-986a-960f34c0c1d1.PNG)

**Comment by vinukumar-vs** [comment]
@aniket-tarento the HTML content should be responsive to fit to any view. In the above snapshot I can see the height & width of the HTML is bigger than the available width & height of the content-player. Hence you can see the scrollbars.

You can try the HTML content alone opening in the browser(index.html) & resize or use "toggle device toolbar" of the chrome to check is HTML content is responsive or not.

@sharathkashyap

**Reply by aniket-tarento** [reply]
@vinukumar-vs Thanks for the clarification.
